backend/app/src/anonymous-user/anonymous-user.controller.ts

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable max-statements */
+/* eslint-disable max-classes-per-file */
+import {
+	Body,
+	Controller,
+	Post,
+	UseGuards,
+	Req,
+	Logger
+} from "@nestjs/common";
+import
+{
+	IsNotEmpty,
+	IsUUID
+}	from	"class-validator";
+import
+{
+	AnonymousUserService,
+}	from "./anonymous-user.service";
+import
+{
+	AnonymousUserLoginResponseModel,
+	AnonymousUserRegisterResponseModel,
+	AnonymousUserVerifyTokenResModel,
+	CustomRequest
+}	from "./anonymous-user.interface";
+import { AuthorizationGuard } from "./anonymous-user.authorizationGuard";
+
+class AnonymousRegisterDto
+{
+	@IsUUID()
+	@IsNotEmpty()
+	uuid: string;
+}
+
+class AnonymousUserLoginDto
+{
+	@IsUUID()
+	@IsNotEmpty()
+	uuid: string;
+	@IsNotEmpty()
+	password: string;
+}
+
+@Controller("anonymous-user")
+export class AnonymousUserController
+{
+	private readonly logger;
+	constructor(private readonly anonymousUserService: AnonymousUserService)
+	{
+		this.logger = new Logger("anonymous-user controller");
+	}
+
+	@Post("register")
+	getAnonymousRegister(@Body() body:AnonymousRegisterDto)
+	: AnonymousUserRegisterResponseModel
+	{
+		this.logger
+			.log("A user request 'register' route with uid :" + body.uuid);
+		return (this.anonymousUserService.register(body.uuid));
+	}
+
+	@Post("login")
+	anonymousUserLogin(@Body() body: AnonymousUserLoginDto)
+	: AnonymousUserLoginResponseModel
+	{
+		this.logger
+			.log("A user request 'login' route with uid :" + body.uuid);
+		return (this.anonymousUserService.login(body.uuid, body.password));
+	}
+
+	@Post("verify-token")
+	@UseGuards(AuthorizationGuard)
+	verifyToken()
+	: AnonymousUserVerifyTokenResModel				// login or clear, cause user already have an uuid
+	// but seams to be an error, user are trigger without
```

Comment:
seems*
If it seems to be an error, what should we do ? Something to change it or make an issue to remember to change it maybe ?

Reply:
seems fixed with new Component front/app/src/Component/ConnectionState/DisplayAnonymousConnection.tsx
previous front/app/src/Component/ConnectionState/DisplayAnonymousConnect.tsx

we can safely remove this comment.